transformations/mixed_language_perturbation/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "mixed_language_perturbation",
+  "test_cases": [
+    {
+      "class": "MixedLanguagePerturbation",
+      "inputs": {
+        "sentence": "Andrew finally returned the Comic book to Chris that I bought last week"
+      },
+      "outputs": {
+        "sentence": "Andrew finally returned Die Comic book to Chris that I bought last Woche"
```

Comment:
It seems like all the code-mixed words are by default in uppercasing. I guess that's caused by the translation model, which would always uppercase the first input. I think the perturbation would be more natural if the casing for the original word and the code-mixed word is the same. For example, "the" -> "die", "Comic" -> "Komik".

Reply:
Good point! We followed your suggestion and pushed it in our latest commit :)